bin/station.js

```diff
@@ -7,11 +7,19 @@ import * as saturnNode from '../lib/saturn-node.js'
 import { fileURLToPath } from 'node:url'
 import { createLogStream } from '../lib/log.js'
 
+const repoRoot = join(dirname(fileURLToPath(import.meta.url)), '..')
+
 const {
   FIL_WALLET_ADDRESS,
   XDG_STATE_HOME = join(homedir(), '.local', 'state')
 } = process.env
 
+if (process.argv.includes('-v') || process.argv.includes('--version')) {
+  const pkg = await fs.readFile(join(repoRoot, 'package.json'), 'utf8')
+  console.log(JSON.parse(pkg).version)
```

Comment:
Do we want to print versions of the installed modules in the future? If yes, do we want to make this output forward-compatible by including the package name in the output too?

```suggestion
  const meta = JSON.parse(pkg)
  console.log("%s: %s", meta.name, meta.version)
```

Reply:
great idea!